### PR TITLE
Inbox item updates: shorter date, orange dot on side, no orange ring

### PR DESCRIPTION
**Pull request by chrisnojima**
This updates our inbox items to match the new zeplin designs. This is more complex than it could be due to avatar crappiness. Something we should cleanup later.

@keybase/react-hackers

### CODE REVIEW

File: shared/chat/conversations-list/index.desktop.js [hunk 1]
```diff
@@ -11,30 +11,28 @@ import type {InboxState} from '../../constants/chat'
...
+    </div>
   </div>
 )
 
-const rowBorderColor = (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) => {
+function rowBackgroundColor (hasUnread: boolean, isSelected: boolean) {
+  return isSelected ? globalColors.white : hasUnread ? globalColors.darkBlue : globalColors.darkBlue4
+}
+
+// All this complexity isn't great but the current implmentation of avatar forces us to juggle all these colors and
```
Comment (cjb) [inline shared/chat/conversations-list/index.desktop.js]: implementation

File: shared/chat/conversations-list/index.desktop.js [hunk 2]
```diff
@@ -11,30 +11,28 @@ import type {InboxState} from '../../constants/chat'
...
+// forces us to explicitly choose undefined/the background/ etc. This can be cleaned up when avatar is simplified
+function rowBorderColor (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) {
   // Not the most recent? Don't color
-  if (idx !== lastParticipantIndex) {
```
Comment (cjb) [inline shared/chat/conversations-list/index.desktop.js]: update the comment above?

File: shared/chat/conversations-list/index.desktop.js [hunk 3]
```diff
@@ -11,30 +11,28 @@ import type {InboxState} from '../../constants/chat'
...
+  if (idx === lastParticipantIndex) {
```
Comment (MarcoPolo) [inline shared/chat/conversations-list/index.desktop.js]: What if this function took `isLastParticipant: boolean` and `isOnlyParticipant`. That would simplify this function a bit, no?

File: shared/common-adapters/multi-avatar.js [hunk 4]
```diff
@@ -22,11 +22,11 @@ const MultiAvatar = ({avatarProps, singleSize, multiSize, style}: Props) => {
     return null
   }
 
-  const leftProps: AvatarProps = avatarProps[0]
-  const rightProps: AvatarProps = avatarProps[1]
+  const leftProps: AvatarProps = avatarProps[1]
+  const rightProps: AvatarProps = avatarProps[0]
```
Comment (MarcoPolo) [inline shared/common-adapters/multi-avatar.js]: I'm a little confused, what's this change about? `the ol switch-a-roo`?

File: shared/chat/conversations-list/index.desktop.js (hunk 3 above)
Comment (MarcoPolo) [inline shared/chat/conversations-list/index.desktop.js]: or even if it calculated these as `const` and then used them below. I just find L35 kinda hard to understand

File: shared/common-adapters/multi-avatar.js (hunk 4 above)
Comment (chrisnojima) [inline shared/common-adapters/multi-avatar.js]: i think the input avatars should be in priority order, so the first should be on top and second behind, but its also rendered right to left so this enforces that. we were actually drawing it incorrectly before. like

avatarProps = [a,b] 
should render
```
[b]
   [a]
```

**Pull request merged**